lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -113,14 +113,15 @@ private function import($filepath, $sourceName, $indexDuringImport = false)
           $self->columnValue('subjectAuthorizedFormOfName'), ['culture' => $self->columnValue('culture')]);
 
         // Determine type ID of relationship type
-        $relationTypeId = array_search(
+        $relationTypeId = array_search_case_insensitive(
           $self->columnValue('relationType'),
           $self->status['actorRelationTypes'][$self->columnValue('culture')]
         );
 
         if (!$relationTypeId)
         {
-          throw new sfException(sprintf('Unknown relationship type %s:', $self->columnValue('relationType')));
+          $error = sprintf('Unknown relationship type "%s"... skipping row.', $self->columnValue('relationType'));
+          print $self->logError($error);
```

Comment:
Is `print` really needed in here and in L137?

Reply:
Yeah print or echo are needed as the log error function returns the argument prepended with the current CSV row number of the import (kind of unintuitive, alas).